crates/core_arch/src/powerpc/altivec.rs

```diff
@@ -3233,18 +3233,18 @@ mod sealed {
 
     #[unstable(feature = "stdarch_powerpc", issue = "111145")]
     pub trait VectorRl {
-        type B;
-        unsafe fn vec_rl(self, b: Self::B) -> Self;
+        type Shift;
+        unsafe fn vec_rl(self, b: Self::Shift) -> Self;
     }
 
     macro_rules! impl_vec_rl {
         ($fun:ident ($a:ident)) => {
             #[unstable(feature = "stdarch_powerpc", issue = "111145")]
             impl VectorRl for $a {
-                type B = t_u!($a);
+                type Shift = t_u!($a);
```

Comment:
Should this be `Bits` for `B`? that would preserve the initial-letter relationship.

Reply:
It would had been a better name indeed :)